dependencies.yaml

```diff
@@ -4,7 +4,7 @@ files:
     output: conda
     matrix:
       cuda: ["11.8", "12.5"]
-      arch: [x86_64]
+      arch: [x86_64, aarch64]
```

Comment:
@jcrist the PR to merge cuda 12.8 support was just merged, so the `all_cuda...` files will need to be regenerated in this PR before merging

Reply:
Done.